pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -1569,7 +1571,7 @@ def _append_context_managers_to_stack(self, node: nodes.Assign) -> None:
         else:
             assignees = [node.targets[0]]
             values = [node.value]
-        if Uninferable in (assignees, values):
+        if any(isinstance(n, UninferableBase) for n in (assignees, values)):
```

Comment:
Unrelated to this PR, but this looks fishy. Why would Uninferable be in a tuple of lists? 🤔

Reply:
Yeah.. I guess `elts` is `Uninferable` sometimes? Or this is just 100% broken..

Reply:
Would you mind opening a follow up issue to investigate?